src/app/_services/api/uqj.service.ts

```diff
@@ -39,59 +35,25 @@ export class UqjService {
         }
 
         return this.http
-            .get<PaginatedResult<UQJ>>(this.userUqjUrl, {params})
-            .pipe(
-                catchError(
-                    this.handleError<PaginatedResult<UQJ>>(
-                        `getAllUserUQJ`
-                    )
-                )
-            );
+            .get<PaginatedResult<UQJ>>(url, {params})
+            .pipe(catchError(this.apiService.handleError<PaginatedResult<UQJ>>(`Error occurred while fetching UQJs`)));
```

Comment:
The error is displayed to the user `UQJ` is not something that users understand. Don't change it to `User Question Junctions`. Put something that is meaningful to the end user.